modules/web/src/app/project/component.ts

```diff
@@ -546,7 +546,7 @@ export class ProjectComponent implements OnInit, OnChanges, OnDestroy {
     component.projectId = projectId;
     component.showIcon = this.showCards;
     component.showAsCard = false;
-    component.showDetailsOnHover = false;
+    component.showDetailsOnHover = true;
```

Comment:
Empty card when quotas are not configured.


https://user-images.githubusercontent.com/17727069/210220963-33175ac2-9acb-4296-99e5-2a08ac90beaa.mp4

Reply:
As per this task _"Fix spacing between owner, role row and quota row, it should be the same as the space between cluster and owner"_ i see space is equalised but i thin the space between  bars or "No quotas configured" has been reduced may be slight **padding** can be better this is my personal opinion compare to previous version which have more padding at bottom.

wdyt?